src/Engine/movement_handler.py

```diff
@@ -4,7 +4,23 @@
 
 
 def move_player(player: Player, key_depressed, movement_distance_x, movement_distance_y, sound_effect):
-    """Handles player movement"""
+    """
```

Comment:
Yeah this kind of documentation definitely needed to be done lol. Great job on it!